requirements.txt

```diff
@@ -80,72 +74,63 @@ drf-amsterdam==0.4.1
     # via -r requirements.in
 drf-extensions==0.7.1
     # via drf-amsterdam
-drf-yasg==1.21.5
+drf-yasg==1.21.6
     # via -r requirements.in
 et-xmlfile==1.1.0
     # via openpyxl
 idna==3.4
     # via requests
 inflection==0.5.1
     # via drf-yasg
-itypes==1.2.0
-    # via coreapi
-jinja2==3.1.2
-    # via coreschema
-jwcrypto==1.4.2
+jwcrypto==1.5.0
     # via datapunt-authorization-django
 markuppy==1.14
     # via tablib
-markupsafe==2.1.2
-    # via jinja2
 odfpy==1.4.1
     # via tablib
 openpyxl==3.1.2
     # via tablib
-packaging==23.0
+packaging==23.1
     # via drf-yasg
-psycopg2==2.9.5
+psycopg2==2.9.6
     # via -r requirements.in
 pycparser==2.21
     # via cffi
 python-dateutil==2.8.2
     # via django-datetime-utc
-pytz==2023.2
+pytz==2023.3
     # via
     #   -r requirements.in
     #   django
     #   djangorestframework
     #   drf-yasg
 pyyaml==6.0
-    # via tablib
-requests==2.28.2
+    # via
+    #   drf-yasg
+    #   tablib
+requests==2.31.0
     # via
     #   -r requirements.in
-    #   coreapi
     #   datapunt-authorization-django
-ruamel-yaml==0.17.21
-    # via drf-yasg
-ruamel-yaml-clib==0.2.7
-    # via ruamel-yaml
```

Comment:
Zijn deze dependencies voor ons obsolete geworden? Met andere woorden, gebruiken we deze niet meer?